git-patches/zostagging/zos-tag.h.patch

```diff
@@ -0,0 +1,83 @@
+diff --git a/src/zos-tag.h b/src/zos-tag.h
+new file mode 100644
+index 0000000..58f3b67
+--- /dev/null
++++ b/src/zos-tag.h
+@@ -0,0 +1,77 @@
++#ifndef __ZOS_TAG__
++  #define __ZOS_TAG__ 1
++
++#ifdef __MVS__
++/* See: https://www.ibm.com/docs/en/zos/latest?topic=lf-toccsid-convert-codeset-name-coded-character-set-id */
++
++static void chgpfx(char* encoding, size_t enclen, size_t pfxlen, const char* normpfx, size_t normpfxlen)
```

Comment:
Minor: Are they picking with the spacing? Seems their convention is to have a space after the function name
```suggestion
+static void chgpfx (char* encoding, size_t enclen, size_t pfxlen, const char* normpfx, size_t normpfxlen)
```